docs/configuration-validation.md

```diff
@@ -17,3 +17,10 @@ If you were presented with configuration error (or a warning, depending on `conf
 **Note**: In a warning mode (with `configValidationMode: warn` set in configuration) Framework commands are not blocked in any way, e.g. `sls deploy` will still attempt to deploy the service normally (still depending on the source of the warning, success of a deployment may vary)
 
 If you find this functionality problematic, you may also turn it off with `configValidationMode: off` setting.
+
+## Supported Values
```

Comment:
Thanks for the suggestion @FinchPowers - instead of listing supported values here, I think it would be better to add a sentence at the beginning of the section (after first sentence) that supported values are `error`, `warn` and `off. The table below kinda repeats explanations that is already mentioned above and is not entirely valid (it's not about failing packaging).